packages/content/types/parsers/yaml/index.d.ts

```diff
@@ -0,0 +1,10 @@
+export interface IYAML {
```

Comment:
options can be typed

Reply:
should be class

Reply:
Updated it to a class with correct options according to documenation